docs/UserGuide.md

```diff
@@ -106,7 +304,28 @@ Examples:
 * `listc s/points` lists all customers sorted by points
 * `listc f/marked` lists all bookmarked customers. 
 
-### View a customer : `viewc`
+#### Locating customers by name : `findc`
+
+Finds customers whose names contain any of the given keywords.
+
+Format: `findc KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+* Only the name is searched.
+* Only full words will be matched e.g. `Han` will not match `Hans`
+* Customers matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+
+Examples:
+* `findc John` returns `john` and `John Doe`
+* `findc alex david` returns `Alex Yeoh`, `David Li`<br>
+
+
+  ![result for 'findc alex david'](images/findAlexDavidResult.png)
+
+
+#### View a customer : `viewc`
 
 Display a customer's information in the information panel.
 
```

Comment:
small typo at line 335 that results in the following visual bug
![image](https://user-images.githubusercontent.com/88615976/228869072-8dcec9ff-f76c-4aa7-8b3c-2b8eb2f5f5ee.png)